### ISSUE TITLE
In line comments should have 2 spaces (not 1)

### ISSUE THREAD

**Issue by giampaolo** [spulec/pep8ify#15]
pep8ify makes the following unnecessarily translation:
- foo  # bar
- foo # bar

As per PEP8 inline comments should be separated by _2_ spaces, not _1_, therefore this particular translation should be avoided.

**Comment by spulec** [comment]
Hi Giampaolo,

Can you give a little bit more specific example? If you look at the input test [here](https://github.com/spulec/pep8ify/blob/master/tests/fixtures/whitespace_before_inline_comment/whitespace_before_inline_comment1_in.py) with the output [here](https://github.com/spulec/pep8ify/blob/master/tests/fixtures/whitespace_before_inline_comment/whitespace_before_inline_comment1_out.py), you can see that this should be happening. It's certainly possible we're missing a case though.

**Comment by giampaolo** [comment]
It seems you're right. Sorry for wasting your time.

**Comment by spulec** [comment]
No worries.

**Comment by giampaolo** [comment]
Ah no sorry, after all it seems I was right.
Here:

```
-    url(r'^group/delete$', views.group_delete,  # crud
+    url(r'^group/delete$', views.group_delete, # crud
```

The two spaces were erroneously turned into 1.

**Comment by spulec** [comment]
Thanks Giampaolo. It looks like the issue was related to inline comments. I believe it should be fixed with the referenced commit.

**Comment by giampaolo** [comment]
I think that introduced a regression. See this:

```
-# -*- coding: UTF-8 -*-
+  # -*- coding: UTF-8 -*-
```

pep8ify should NOT add 2 spaces if line starts with "#"

**Comment by spulec** [comment]
You are correct. It should be fixed with 664daf3c78c68935e472778bc52ee5d67fd0c03f
